infrastructure/development/dns.rf

```diff
@@ -0,0 +1,32 @@
+# DNS Zone
```

Comment:
there's a typo in the extension of this file `rf` -> `tf`

Reply:
fixed in  commit 4c0d84fc78450d69792dd92ff546958890991675